pcsx2/gui/FrameForGS.cpp

```diff
@@ -615,6 +623,22 @@ void GSFrame::OnUpdateTitle( wxTimerEvent& evt )
 	out << std::fixed << std::setprecision(2) << fps;
 	OSDmonitor(Color_StrongGreen, "FPS:", out.str());
 
+	int new_width, new_height;
+	GSgetImageSize(&new_width, &new_height);
+	if (new_width > 0 && new_height > 0)
```

Comment:
With an older GSdx version, `new_width` and `new_height` will be uninitialised, so you might have random dimensions.